### ISSUE TITLE
dwc:month examples have text, do I assume we really only want the numeric?

### ISSUE THREAD

**Issue by debpaul** [tdwg/dwc-qa#197]
Greetings @tdwg/dwc 

See examples for term dwc:month
Examples: `1 (January). 10 (October).`
- [x] I'm guessing we really only want the numeric, not the text string? yes?
- [x] Suggest we add Comment, as in  
Comment:  For a given month, example for January, enter only a `1` in this field, or for February a `2`.

- [ ] OR do we want always two digits (`01`)?

And then in the 
Example: 1

**Comment by ben-norton** [comment]
Only the numeric representation please. Numeric months are readily machine readable and globally accessible.

**Comment by tucotuco** [comment]
These are worthy suggestions and I welcome a term change request for them. We do not want to suggest 01 as that suggests a string rather than a number even if in most encoding schemes the value would be text. We could go all out and have:
Example: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 (and no other values)

**Comment by ben-norton** [comment]
I support the all out approach.

**Comment by debpaul** [comment]
@tucotuco since this is just a change to the example, we don't need to do anything else, right? Just edit the text?

**Comment by tucotuco** [comment]
No, we decided after the 2021 public review that even non-normative changes that are not errata should go through public review because the Maintenance Group alone might not have enough insight across the diverse panorama of Darwin Core usage. These are not errata. The bonus is that it improves awareness of the changes.

**Comment by debpaul** [comment]
So @tucotuco do you want to create the review ticket? Or would you rather I try to do it? If you want me to do it, please point me here to where I do to a) do this and b) the format? Thanks!

**Comment by tucotuco** [comment]
@debpaul  I could do it easily, but it might not be a bad idea to use you to test if the documentation is clear and simple. Let me know if that is OK. [This document](https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md), linked to from the [main Darwin Core GitHub page](https://github.com/tdwg/dwc) and we have [this open issue](https://github.com/tdwg/dwc/issues/436) about making it more obvious how to contribute directly from the most popular document, the [Quick Reference Guide](https://dwc.tdwg.org/terms/).

**Comment by debpaul** [comment]
> @debpaul I could do it easily, but it might not be a bad idea to use you to test if the documentation is clear and simple. Let me know if that is OK. [This document](https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md), linked to from the [main Darwin Core GitHub page](https://github.com/tdwg/dwc) and we have [this open issue](https://github.com/tdwg/dwc/issues/436) about making it more obvious how to contribute directly from the most popular document, the [Quick Reference Guide](https://dwc.tdwg.org/terms/).

@tucotuco i will try — good practice.

**Comment by MattBlissett** [comment]
Isn't this already OK?

http://rs.tdwg.org/dwc/terms/month

> `1` (January). `10` (October).

Many of the examples show pink fixed-width example values and black proportional-width explanations.

**Comment by ymgan** [comment]
The current definition as @MattBlissett mentioned is clear to me. However, I still get datasets with zero padded months (e.g. `01`, `02`, `03` etc) from data providers who are familiar with Darwin Core. 

So I support the all out approach as @tucotuco suggested.

**Comment by debpaul** [comment]
> Isn't this already OK?
> 
> http://rs.tdwg.org/dwc/terms/month
> 
> > `1` (January). `10` (October).
> 
> Many of the examples show pink fixed-width example values and black proportional-width explanations.

@MattBlissett Hm. (Probably just me then, but I found it confusing as if I could put `1 (Janurary)` in the field. I see the other terms you are referring to that use a similar strategy. I don't know what's best here. We could also just say in the Comment: Please do not put leading zeroes.
